### DISCUSSION TITLE
How should I modify the spacing between the operation buttons

### DISCUSSION THREAD

**Discussion by zhushenhang** [material-table-core/core#61] [Q&A]
When I use padding="dense", how should I modify the spacing between the operation buttons?
![2C546210-41EA-4e79-A5C2-3A8A664C0437](https://user-images.githubusercontent.com/24982716/104556513-b2c22980-567a-11eb-9bad-aeeaf2b5e0ed.png)
I want to make the distance between my edit button and delete button bigger.

**Comment by matthewoestreich** [comment]
Hey would you mind opening an issue with a reproducible demo? The short answer is to override that component

**Reply by zhushenhang** [reply]
https://codesandbox.io/s/awesome-taussig-vbs5h?file=/src/App.js  
This is my demo , but this csb has an error .I am not very good at using csb.
Just look at the code.
thanks.
